src/libs/extra/source.liq

```diff
@@ -284,8 +287,9 @@ def source.cue(
   ~comment=null(),
```

Comment:
It would be nice to have `file_type` default to file extension when this is defined instead of hardcoding mp3

Reply:
Done